File: FireSaverApi/Dtos/BuildingDtos/BuildingInfoDto.cs
using System.Collections.Generic;
using FireSaverApi.DataContext;
using FireSaverApi.Dtos.CompartmentDtos;

namespace FireSaverApi.Dtos.BuildingDtos
{
    public class BuildingInfoDto
    {
        public int Id { get; set; }
        public IList<UserInfoDto> ResponsibleUsers { get; set; }
        public IList<FloorDto> Floors { get; set; }
        public string Address { get; set; }
        public string Info { get; set; }
        public PositionDto BuildingCenterPosition { get; set; }
        public IList<ShelterDto> Shelters { get; set; }
        public double? SafetyDistance { get; set; }
    }
}

File: FireSaverSPA/src/app/Models/BuildingService/buildingInfoDto.ts
import { FloorDto } from "../Compartment/floorDto";
import { Postion } from "../PointService/pointDtos";
import { UserInfoDto } from "../UserService/userInfoDto";
import { ShelterDto } from "./ShelterDto";

export class BuildingInfoDto {
    id: number;
    responsibleUsers: UserInfoDto[]
    floors: FloorDto[]
    shelters: ShelterDto[]
    address: string;
    info: string;
    buildingCenterPosition: Postion;
    region:String;
}